src/main/java/org/apache/sysds/runtime/instructions/InstructionUtils.java

```diff
@@ -420,7 +421,11 @@ else if ( opcode.equalsIgnoreCase("uacmin") ) {
 			AggregateOperator agg = new AggregateOperator(Double.POSITIVE_INFINITY, Builtin.getBuiltinFnObject("min"));
 			aggun = new AggregateUnaryOperator(agg, ReduceRow.getReduceRowFnObject(), numThreads);
 		}
-		
+		else if ( opcode.equalsIgnoreCase("uacd") || opcode.equalsIgnoreCase("uacdr")
+				|| opcode.equalsIgnoreCase("uacdc") ) {
```

Comment:
Should we add the approximate methods as well?

Reply:
I have added them in a separate "section" for sketch operators:
```
		String2SPInstructionType.put( "uacd"    , SPType.AggregateUnary);
		String2SPInstructionType.put( "uacdr"   , SPType.AggregateUnary);
		String2SPInstructionType.put( "uacdc"   , SPType.AggregateUnary);

		// Aggregate unary sketch operators
		String2SPInstructionType.put( "uacdap" , SPType.AggregateUnarySketch);
		String2SPInstructionType.put( "uacdapr", SPType.AggregateUnarySketch);
		String2SPInstructionType.put( "uacdapc", SPType.AggregateUnarySketch);
```